packages/docs/src/pages/en/getting-started/installation.md

```diff
@@ -37,6 +37,12 @@ Now that you have an instantiated project, you can add the Vuetify [Vue CLI pack
 vue add vuetify
 ```
 
+<alert type="warning">
+
+  This command will make changes to your `App.vue`, components folder, vue.config.js, etc. If you are adding vuetify to an existing project, please make sure you  commit your code to avoid any data loss. Please refer to the `Webpack Install` section to install in existing projects.
```

Comment:
The webpack install section isn't relevant for vue-cli. The template changes can also be skipped if you select the advanced install option after running `vue add vuetify`, not sure if that should be mentioned here though.

Reply:
Its probably worth mentioning